compiler/tests-compiler/direct_calls.ml

```diff
@@ -123,53 +123,91 @@ let%expect_test "direct calls with --enable effects" =
            M1.f 1; M2.f 2
 |}
   in
-  print_fun_decl code (Some "test1");
-  print_fun_decl code (Some "test2");
-  print_fun_decl code (Some "test3");
-  print_fun_decl code (Some "test4");
+  print_fun_decl code (Some "test1$0");
+  print_fun_decl code (Some "test1$1");
+  print_var_decl code "test1";
```

Comment:
Maybe a helper could extract names to lookup automatically by looking up `test1`, and collecting arguments to `caml_cps_closure`.

Reply:
Done in https://github.com/ocsigen/js_of_ocaml/pull/1461/commits/2360daaae0be44740247de1a188302500b46f8af.